### DISCUSSION TITLE
Feature request: Window Icon before window title

### DISCUSSION THREAD

**Discussion by Rajaniraiyn** [nextapps-de/winbox#135] [Ideas]
Hi!
   I really loved your project I have planned to integrate this within [my project](https://github.com/Rajaniraiyn/windows11). We have implemented a similar window, but it is not much feature rich and extensible like yours.

Please add an API to add icon before title. that will be make this more good.

**Comment by mossaiby** [comment]
Hi @Rajaniraiyn I think this is possible using CSS if you use flexbox.

**Comment by Rajaniraiyn** [comment]
Yes we can do so. but if it is on this project natively then it will be very helpful for others too.

**Comment by rvanroon** [comment]
I've used the following code to add a font-awesome icon to the titlebar after initiating a Winbox:

```
var winbox = new WinBox( { ...your settings ... } );
$("#" + winbox.id + " .wb-title").prepend("<i class='fa fa-radiation pr-2'></i>"); 
```

**Comment by Rajaniraiyn** [comment]
That's cool idea. But it will be far more useful when we have it natively in WinBox with more features

**Comment by ts-thomas** [comment]
Custom icon for the heading window toolbar is now supported.
